test/fixtures/rooms.yml

```diff
@@ -6,29 +6,34 @@ one:
   description: Room n°1
   site_id: 1
   display_on_home_page: true
+  status: :active
```

Comment:
Is this necessary if it's the default value?

Reply:
no indeed